mscore/startcenter.h

```diff
@@ -18,7 +18,7 @@
 
 namespace Ms {
 
-#if 0
+#if USE_WEBENGINE
```

Comment:
For other similar ones (USE_ALSE, HAS_MIDI, USE_PULSEAUDIO, USE_LAME, ...) we use `#ifdef`. It does work with `#if' too though